src/Dispatcher.jl

```diff
@@ -35,6 +35,7 @@ using IterTools
 using LightGraphs
 using Memento
 using ResultTypes
+using Distributed
```

Comment:
To maintain 0.6 support, this should instead be `using Compat.Distributed`, with the minimum Compat requirement in REQUIRE set sufficiently new.

Reply:
Yup, I mostly put this branch up to be easier to pull down to my 0.6 repo for adding compats